app/assets/javascripts/admin/addon/components/admin-config-areas/about.gjs

```diff
@@ -35,7 +39,10 @@ export default class AdminConfigAreasAbout extends Component {
     };
   }
 
-  saveCallback() {}
+  @action
+  setSavingStatus(status) {
```

Comment:
This is interesting, I think this is a good way of handling it, maybe at some point it makes sense to add some generic service for this so you could do something like `this.configAreaService.setSaving("about")` then do `this.configAreaService.getSavingStatus("about")` in child components. Anyway that's a thought for the future